File: src/0004-Median of Two Sorted Arrays/Solution.php
<?php

class Solution
{
    /**
     * @param Integer[] $nums1
     * @param Integer[] $nums2
     * @return Float
     */
    function findMedianSortedArrays($nums1, $nums2)
    {
        $arr = array_merge($nums1, $nums2);
        //print_r($arr);
        sort($arr);
        $len = sizeof($arr);
        //print_r($arr);
        if ($len % 2 == 0) {
            return ($arr[$len / 2] + $arr[$len / 2 - 1]) / 2;
        }

        return $arr[($len - 1) / 2];
    }
}

$nums1 = [1, 2, 5, 7, 8, 12, 32];
$nums2 = [3, 4, 8, 9, 11, 13, 22, 89];

$s = new Solution();
print_r($s->findMedianSortedArrays($nums1, $nums2));

File: src/0004-Median of Two Sorted Arrays/Solution.java
/*
import com.sun.tools.javac.util.ArrayUtils;

import java.util.Arrays;

class Solution {
    public double findMedianSortedArrays(int[] nums1, int[] nums2) {
        int[] arr = ArrayUtils.addAll(nums1, nums2);
        Arrays.sort(arr);
        int len = arr.length;
        double res = 0;
        if (len % 2 == 0) {
            res = (arr[len / 2] + arr[len / 2 - 1]) / 2;
        } else {
            res = arr[(len - 1) / 2];
        }

        return res;
    }
}*/
